Documentation/api.md

```diff
@@ -204,6 +204,7 @@ Endpoint defines a scrapeable endpoint serving Prometheus metrics.
 | bearerTokenFile | File to read bearer token for scraping targets. | string | false |
 | bearerTokenSecret | Secret to mount to read bearer token for scraping targets. The secret needs to be in the same namespace as the service monitor and accessible by the Prometheus Operator. | [v1.SecretKeySelector](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.11/#secretkeyselector-v1-core) | false |
 | honorLabels | HonorLabels chooses the metric's labels on collisions with target labels. | bool | false |
+| honorTimestamps | HonorTimestamps allows to disable honoring timestamps | *bool | false |
```

Comment:
But honor_timestamps should be true by default, why is this false?

Reply:
+1 Prometheus has true by default...

Reply:
By default this should be `nil` to preserve backwards compatibility.

Reply:
Also, this is `false` because generator made it this way :P